.github/actions/fling-test/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: Test-flinging
```

Comment:
Seeing this, best to just place this in https://github.com/canonical/testflinger so that testflinger is self contained, and we don't need to be clever about naming the job (e.g. `https://github.com/canonical/testflinger/.github/actions/submit.yaml`)